src/lengthCodes.ts

```diff
@@ -0,0 +1,36 @@
+export function getLengthCodeForObjectSize(objectSize: number): number {
```

Comment:
Could be nice if LengthCode was a type? Since it is not actually a freeform number but a specific known set of codes?